virt/virt-4-11-release-notes.adoc

```diff
@@ -181,10 +181,26 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 [id="virt-4-11-known-issues"]
 == Known issues
 
-//New known issues for 4.11 go above this comment
+* Some persistent volume claim (PVC) annotations created by the Containerized Data Importer (CDI) can cause the virtual machine snapshot restore operation to hang indefinitely. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2070366[*BZ#2070366*])
+** As a workaround, you can remove the annotations manually:
+. Obtain the xref:../virt/virtual_machines/virtual_disks/virt-managing-vm-snapshots.adoc#virtual-machine-snapshot-controller-and-custom-resource-definitions-crds[VirtualMachineSnapshotContent] custom resource (CR) name from the `status.virtualMachineSnapshotContentName` value in the `VirtualMachineSnapshot` CR.
+. Edit the `VirtualMachineSnapshotContent` CR by removing all lines that contain `k8s.io/cloneRequest`.
+. Optional: Delete the `DataVolume` and `PersistentVolumeClaim` objects in this namespace if their name begins with `restore-` and they are not referenced by virtual machines. This step is required if you did not specify a value for `spec.dataVolumeTemplates` in the `VirtualMachine` object.
```

Comment:
Nitpicky suggestion (take or leave it) because I'm not sure if people will read the whole step if it starts with "Optional":

```suggestion
. If you did not specify a value for `spec.dataVolumeTemplates` in the `VirtualMachine` object, delete any `DataVolume` and `PersistentVolumeClaim` objects in this namespace where both of the following conditions are true:
* The object's name begins with `restore-`.
* The object is not referenced by virtual machines. 
+
This step is optional if you specified a value for `spec.dataVolumeTemplates`.
```